config.py

```diff
@@ -13,6 +13,11 @@
         'auto-approve': {
             'search_name': None},
         'auth-endpoint': 'https://oauth.wildapricot.org/auth/token',
+        'cancel': {
```

Comment:
Much of this isn't used. I was holding more information in the config and decided against it. I'll remove these extra fields.